doc/pick_place/pick_place_tutorial.rst

```diff
@@ -0,0 +1,46 @@
+Pick and Place Tutorial
+============================
+
+In MoveIt!, grasping is done using the move group interface. In order to grasp an object we need to create ``moveit_msgs::Grasp`` msg which will allow defining the various poses and postures involved in a grasping operation.
+Watch this video to see the output of this tutorial:
+
+.. raw:: html
+
+    <div style="position: relative; padding-bottom: 5%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe width="700px" height="400px" src="https://www.youtube.com/embed/cHxkQwSyrgY?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
+Running The Demo
+----------------
+Open two shells. In the first shell start RViz and wait for everything to finish loading: ::
+
+    roslaunch panda_moveit_config demo.launch
+
+In the second shell run the pick and place tutorial: ::
+    
+    rosrun moveit_tutorials pick_place_tutotrial
```

Comment:
You need a roslaunch for this

Reply:
i had initially created a launch but the problem is that we need to wait for everything to load before running this node.